### PR TITLE
Granquet/config account sort

### PR DESCRIPTION
**Pull request by granquet**

### CODE REVIEW

File: src/conf.rs [hunk 1]
```diff
@@ -239,12 +242,15 @@ impl AccountConf {
     }
 }
 
+use std::str::FromStr;
```
Comment (granquet) [inline src/conf.rs:245]: oops, this is unused, this is leftover from the previous version.

File: src/components/mail/listing.rs [hunk 2]
```diff
@@ -23,6 +23,7 @@ use super::*;
 use crate::conf::accounts::JobRequest;
 use crate::types::segment_tree::SegmentTree;
 use melib::backends::EnvelopeHashBatch;
+use melib::conf::AccountSettings;
```
Comment (epilys) [inline src/components/mail/listing.rs:26]:
```suggestion
```

File: src/conf.rs [hunk 3]
```diff
@@ -877,7 +884,12 @@ mod pp {
             for theme_mailbox in xdg_dirs.find_config_files("themes") {
                 let read_dir = std::fs::read_dir(theme_mailbox)?;
                 for theme in read_dir {
-                    ret.push_str(&pp_helper(&theme?.path(), 0)?);
+                    let file = theme.unwrap().path();
```
Comment (epilys) [inline src/conf.rs:887]:
```suggestion
                    let file = theme?.path();
```

**Review by epilys — CHANGES_REQUESTED** [review]

**Comment by epilys** [comment]
Looks good to me other than those really minor details! Ready to merge when these are fixed.

**Review by epilys — APPROVED** [review]

**Pull request merged**